pkg/auth/iam/iam.go

```diff
@@ -60,6 +62,11 @@ type ErrorResponse struct {
 
 // NewFilter creates new Filter instance
 func NewFilter(client iam.Client) *Filter {
+	enableSubdomainFiltering, _ := strconv.ParseBool(os.Getenv(constant.EnvEnableSubdomainFiltering))
```

Comment:
Can we make it consistent using the LookUpEnv so we also able to handle if env not exists also?

Something like this
`if s, exists := os.LookupEnv("FULL_ACCESS_LOG_ENABLED"); exists {
		value, err := strconv.ParseBool(s)
		if err != nil {
			logrus.Errorf("Parse FULL_ACCESS_LOG_ENABLED env error: %v", err)
		}
		FullAccessLogEnabled = value
	}`